package.json

```diff
@@ -14,6 +14,8 @@
   },
   "dependencies": {
     "@react-native-community/masked-view": "0.1.6",
```

Comment:
Can you provide some info about all these packages? What does each of them bring us?

(The more packages we have, the heavier our app in the end. And it also brings more maintenance cost)